src/main/java/edu/kis/powp/jobs2d/command/history/HistoryEntry.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.command.history;
+
+public class HistoryEntry {
+
+    private String date;
+    private String commandName;
```

Comment:
There lacks data to reinstate a previous command (it looks more like a log, but I expected an active history). I would like such an option to be implemented also from the GUI.